<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProviderTab</class>
 <widget class="QWidget" name="ProviderTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>387</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="mainGridLayout">
   <property name="spacing">
    <number>9</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="DataInteractionToolBar" name="dataInteractionToolBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="DataFrame" name="dataListFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="dataListTitle">
        <property name="text">
         <string>PROVIDER LIST</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="dataListView">
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="dataEditionWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>7</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>9</number>
      </property>
      <item row="1" column="0">
       <spacer name="confirmationHorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="WidgetGroup" name="confirmationWidgetGroup" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="confirmationOkPushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="confirmationCancelPushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QScrollArea" name="dataEditionScrollArea">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="dataEditionScrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>507</width>
           <height>312</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>9</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="DataFrame" name="idDataFrame">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="idTitle">
               <property name="text">
                <string>PROVIDER ID</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="waybillIdWidget" native="true">
               <layout class="QGridLayout" name="gridLayout_54">
                <item row="2" column="1">
                 <widget class="WidgetGroup" name="waybillIdWidgetGroup" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="waybillIdLabel">
                     <property name="text">
                      <string>Waybill ID</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="waybillIdLineEdit">
                     <property name="frame">
                      <bool>false</bool>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="WidgetGroup" name="waybillCountryCodeWidgetGroup" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="waybillCountryCodeLabel">
                     <property name="text">
                      <string>Country code</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="waybillCountryCodeLineEdit">
                     <property name="frame">
                      <bool>false</bool>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
               <zorder>waybillIdWidgetGroup</zorder>
               <zorder>waybillCountryCodeWidgetGroup</zorder>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="DataFrame" name="contractDataFrame">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="contractTitle">
               <property name="text">
                <string>PROVIDER CONTRACTS</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="contractWidget" native="true">
               <layout class="QGridLayout" name="gridLayout_56">
                <item row="0" column="2">
                 <widget class="WidgetGroup" name="waybillTransportVehicleWidget" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_15">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="waybillTransportVehicleLabel">
                     <property name="text">
                      <string>Vehicle</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="waybillTransportVehicleLineEdit">
                     <property name="frame">
                      <bool>false</bool>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="waybillTransportVehiclePushButton">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>24</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>24</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>...</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="WidgetGroup" name="waybillTransportRegistrationNoWidgetGroup" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_14">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="waybillTransportRegistrationNoLabel">
                     <property name="text">
                      <string>Registration n°</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="waybillTransportRegistrationNoLineEdit">
                     <property name="frame">
                      <bool>false</bool>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="0" rowspan="2">
                 <widget class="WidgetGroup" name="waybillTransportMeanImageWidgetGroup" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_13">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="waybillTransportMeanImageLabel">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>72</width>
                       <height>72</height>
                      </size>
                     </property>
                     <property name="sizeIncrement">
                      <size>
                       <width>1</width>
                       <height>1</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="1" rowspan="2">
                 <widget class="WidgetGroup" name="waybillTransportMeanWidgetGroup" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="waybillTransportMeanLabel">
                     <property name="text">
                      <string>Mean</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="waybillTransportMeanComboBox">
                     <property name="currentIndex">
                      <number>6</number>
                     </property>
                     <item>
                      <property name="text">
                       <string>Car</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>Icon/Glyph/Car.png</normaloff>Icon/Glyph/Car.png</iconset>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Bus</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>Icon/Glyph/Bus.png</normaloff>Icon/Glyph/Bus.png</iconset>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Airplane</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>Icon/Glyph/Airplane.png</normaloff>Icon/Glyph/Airplane.png</iconset>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Boat</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>Icon/Glyph/Boat.png</normaloff>Icon/Glyph/Boat.png</iconset>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Animal</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>Icon/Glyph/Dog.png</normaloff>Icon/Glyph/Dog.png</iconset>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Train</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>Icon/Glyph/Train.png</normaloff>Icon/Glyph/Train.png</iconset>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Truck</string>
                      </property>
                      <property name="icon">
                       <iconset>
                        <normaloff>Icon/Glyph/Truck.png</normaloff>Icon/Glyph/Truck.png</iconset>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="DataFrame" name="statusDataFrame">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="statusTitle">
               <property name="text">
                <string>PROVIDER STATUS</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="statusWidget" native="true">
               <layout class="QGridLayout" name="gridLayout_55">
                <item row="0" column="0">
                 <widget class="StatusWidgetGroup" name="statusWidgetGroup" native="true"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataInteractionToolBar</class>
   <extends>QWidget</extends>
   <header>DataInteractionToolBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetGroup</class>
   <extends>QWidget</extends>
   <header>WidgetGroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DataFrame</class>
   <extends>QFrame</extends>
   <header>DataFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StatusWidgetGroup</class>
   <extends>QWidget</extends>
   <header>StatusWidgetGroup.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
